approval_polls/templates/approval_polls/my_polls.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+
+{% block content %}
+  <div  class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+          <div class='well well-sm'>
+            Polls created by {{ user.username }}:
+        </div>
+    </div>
+  </div>
+  {% if latest_poll_list %}
+    <div class='row-fluid top-buffer'>
+      <div class='col-md-12'>
+        {% for poll in latest_poll_list %}
+          <div class='well well-sm'>
+            <a href="{% url 'approval_polls:detail' poll.id %}">{{ poll.question }}</a>
+            &nbsp;&nbsp;&nbsp;<a> remove </a> 
+            &nbsp;&nbsp;&nbsp;<a> suspend </a> 
+            &nbsp;&nbsp;&nbsp;<a> edit </a> 
+            <p>
+              Published on {{ poll.pub_date }} by {{poll.user.username}}
```

Comment:
Remove 'by {{poll.user.username}}' because all polls on this page are the user's polls